assets/src/dashboard/theme.js

```diff
@@ -63,6 +63,7 @@ const theme = {
     shadow: '0px 1px 1px rgba(0, 0, 0, 1)',
   },
   fonts: {
+    googleSans: "'Google Sans', 'Sans Serif'",
```

Comment:
i couldn't decide if this or `fonts.pills` would be better?

Reply:
I think I'd prefer usage-based labels that way should they change we know what we are changing.

Reply:
cool, i'll swap out.